conf/application.conf

```diff
@@ -77,3 +77,10 @@ logger.play=INFO
 
 # Logger provided to your application:
 logger.application=DEBUG
+
+# Enable web analytics for the application.
+# By default analytics is not turned on. Set this property
+# to true and paste the javascript snippet into 'public/analytics/track.js' for
+# enabling web analytics for the application. You may configure an analytics application
+# like piwik. More information on piwik at piwik.org
+enable.analytics=false
```

Comment:
Since ```application.conf``` gets bundled along with the distribution, it is better to include this under ```app-conf```directory which is a separate configuration package. We can set it in ```elephant.conf``` and then pass it to play by editing the start.sh script.